src/pages/AccessPage.tsx

```diff
@@ -110,7 +130,7 @@ const AccessPage = () => {
             <Route path="/signin" component={SignInForm} />
             <Route path="/signup" component={SignUpForm} />
```

Comment:
TODO: We should actually comment our signin and reset, so we'll really only have the two paths, signup and success.